### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by mdussere**
I found few bugs that were blocking me:
 * unknown type MassAnalyzerType
 * wrong usage of get_segmented_scan_from_scan_number
 * using unknow spectrum.masses

### CODE REVIEW

File: fisher_py/raw_file.py [hunk 1]
```diff
@@ -146,8 +146,10 @@ def _get_scan_(self, scan_number: int) -> Tuple[np.ndarray, np.ndarray, np.ndarr
             intensities = np.array(spectrum.intensities)
             charges = np.array(spectrum.charges)
         else:
-            spectrum = self._raw_file_access.get_segmented_scan_from_scan_number(scan_number, None)
-            positions = np.array(spectrum.masses)
+            stats = self._raw_file_access.get_scan_stats_for_scan_number(scan_number)
+            spectrum = self._raw_file_access.get_segmented_scan_from_scan_number(scan_number, stats)
+            mass_ranges = np.array(spectrum.mass_ranges)
```
Comment (dowerner) [inline fisher_py/raw_file.py]: The variable "mass_ranges" is not needed, remove it.

**Comment by dowerner** [comment]
Hello Michael,
Thank you very much for fixing the issues you have found. There is only a minor thing in the code review that needs to be resolved, after that I will merge your PR.
Cheers,
Dominik

**Comment by mdussere** [comment]
Hi Dominik,
I removed the unused variable. Is that OK for you ?

**Review by dowerner — APPROVED** [review]

**Comment by dowerner** [comment]
Looks good to me, will be merged :)
Thank you very much for contributing!

**Pull request merged**